packages/base/src/api/api.ts

```diff
@@ -32,12 +39,18 @@ export function createApi(
 
 export function createAuthenticatedApi(
   username: string,
-  session: CognitoUserSession,
+  accessToken: string,
```

Comment:
Only value needed in session is the `accessToken`, the `session` object doesn't exist anymore in client